Pacifique78/ADC
# #168398069 Ft sign in API database

#### What does this PR do?
Add and test sign-in endpoint with database 

#### Description of Task to be completed
User should be able to log in using the credentials used during sign up period
Whenever a mismatch in data provided occurs, the user will not be able to log in 

#### How should this be manually tested?
See if the user with correct credentials can log in and if the user with any mismatch in credentials will not be allowed to log in. 

#### What are the relevant pivotal tracker stories?
#168398069

#### Screenshots (if appropriate)
![Screenshot (37)](https://user-images.githubusercontent.com/50268181/64601486-50d61d80-d3bd-11e9-844b-58943f70d23a.png)
![Screenshot (38)](https://user-images.githubusercontent.com/50268181/64601487-50d61d80-d3bd-11e9-81cb-26d59098f210.png)
![Screenshot (39)](https://user-images.githubusercontent.com/50268181/64601488-516eb400-d3bd-11e9-9dec-0f94a92927f2.png)
